packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Vertical.Basic.Example.tsx

```diff
@@ -25,6 +25,19 @@ export class VerticalStackBasicExample extends React.Component<{}, {}> {
           <span>Item Three</span>
         </Stack>
 
+        <span>Ordered stack</span>
```

Comment:
Can you also update `Stack.Horizontal.Basic.Example.tsx` with a similar example?